package.json

```
@@ -6,18 +6,22 @@
     "start": "docusaurus start",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy"
+    "deploy": "docusaurus deploy",
+    "postinstall": "patch-package"
```

Isn't this suppose to be like this?
```suggestion
    "postinstall": "yarn patch-package"
```

---

Both are the same assuming the binaries in node modules is in the path.
I just match that one with the one from `patch-package` readme
```diff
 "scripts": {
+  "postinstall": "patch-package"
 }
```
https://github.com/ds300/patch-package#set-up